content/en/docs/zero-code/obi/setup/kubernetes-helm.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Deploy OBI in Kubernetes with Helm
+linkTitle: Helm chart
+description: Learn how to deploy OBI as a Helm chart in Kubernetes.
+weight: 3
+---
+
+{{% alert title="Note" %}}
+
+For more details about the diverse Helm configuration options, check out the
+[OBI Helm chart options](https://github.com/grafana/beyla/blob/main/charts/beyla/README.md)
+document.
+
+{{% /alert %}}
+
+Contents:
+
+<!-- TOC -->
+
+- [Deploying OBI from helm](#deploying-obi-from-helm)
+- [Configuring OBI](#configuring-obi)
+- [Configuring OBI metadata](#configuring-obi-metadata)
+- [Providing secrets to the Helm configuration](#providing-secrets-to-the-helm-configuration)
+<!-- TOC -->
+
+## Deploying OBI from helm
+
+First, you need to add the Grafana helm repository to Helm:
+
+```sh
+helm repo add grafana https://grafana.github.io/helm-charts
+```
+
+The following command deploys a OBI DaemonSet with a default configuration in
+the `beyla` namespace:
```

Comment:
```suggestion
the `beyla` namespace:
```

we need to replace the namespace here

Reply:
In the reworked docs and examples, I'm usually using `obi` as namespace.